src/main/plugins/websearch-plugin/web-search-engine.ts

```diff
@@ -8,4 +8,5 @@ export interface WebSearchEngine {
     priority: number;
     isFallback: boolean;
     encodeSearchTerm: boolean;
+    suggestionUrl: string;
```

Comment:
I would mark this as nullable because not every search engine supports suggestions.

Reply:
Okay.

Reply:
wouldn't that make the tscompiler complain ? since axios or anyother GET method would expect string .

Reply:
Wouldn't that make the tscompiler complain? since axios or any GET method would expect a string.

Reply:
If there is no suggestion url there is no need to fetch suggestions, right?

Reply:
I can resolve empty suggestion for webSearchEngines with no suggestion url.